src/api/config/routes.rb

```diff
@@ -730,16 +730,23 @@ def self.public_or_about_path?(request)
     get 'projects/:project/packages/:package/requests' => 'webui/packages/bs_requests#index', constraints: cons, as: 'packages_requests'
   end
 
+  # project level
+  controller :source_project do
+    get 'source/:project' => :show_project, constraints: cons
+    delete 'source/:project' => :delete_project, constraints: cons
+    post 'source/:project' => :project_command, constraints: cons
+  end
+
   controller :source do
     get 'source' => :index
     post 'source' => :global_command_createmaintenanceincident, constraints: ->(req) { req.params[:cmd] == 'createmaintenanceincident' }
     post 'source' => :global_command_branch,                    constraints: ->(req) { req.params[:cmd] == 'branch' }
     post 'source' => :global_command_orderkiwirepos,            constraints: ->(req) { req.params[:cmd] == 'orderkiwirepos' }
 
     # project level
-    get 'source/:project' => :show_project, constraints: cons
-    delete 'source/:project' => :delete_project, constraints: cons
-    post 'source/:project' => :project_command, constraints: cons
+    # get 'source/:project' => :show_project, constraints: cons
+    # delete 'source/:project' => :delete_project, constraints: cons
+    # post 'source/:project' => :project_command, constraints: cons
```

Comment:
Please just drop these lines. You can take this as a general rule of thumb. If you comment something out because it's not needed anymore you can also just drop:-)